examples/decoding/plot_haxby_anova_svm.py

```diff
@@ -17,8 +17,8 @@
 print('Functional nifti image (4D) is located at: %s' %
       haxby_dataset.func)
```

Comment:
You probably want to use .func[0] here. I am actually surprised that you didn't have to modify .func -> .func[0] in any of the examples.